resources/diameter-s6a/events/pom.xml

```diff
@@ -17,6 +17,11 @@
 			<artifactId>restcomm-slee-ra-diameter-base-common-events</artifactId>
 			<version>${project.version}</version>
 		</dependency>
-	</dependencies>
+    <dependency>
```

Comment:
This dependency should not be required, this comes from an [import which isn't being used](https://github.com/RestComm/jain-slee.diameter/blob/master/resources/diameter-s6a/events/src/main/java/net/java/slee/resource/diameter/s6a/events/avp/PLMNClient.java#L26). Please correct by removing the import and the dependency.